### PR TITLE
Update AUTHORS and mailmap

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by certik** [comment]
+1

**Comment by isuruf** [comment]
@mafiya69 @cbehan @aktech @shivamvats, let me know if there are any changes to be done

File: AUTHORS [hunk 1]
```diff
@@ -17,5 +17,7 @@ Shivam Vats <shivamvats.iitkgp@gmail.com>
 Sumith <sumith1896@gmail.com>
 Sanka Rasnayaka <nuwan.ras@gmail.com>
 Abinash Meher <abinashdakshana999@gmail.com>
-AMiT Kumar <dtu.amit@gmail.com>
+Govind Sahai <gsiitbhu@gmail.com>
+Amit Kumar <dtu.amit@gmail.com>
```
Comment (aktech) [inline AUTHORS]: I am using the capitalised version `AMiT Kumar` now, so this needs to be added to mailmap as well, otherwise it's good +1

Comment (isuruf) [inline AUTHORS]: Which one do you prefer `AMiT Kumar` or `Amit Kumar`?

Comment (aktech) [inline AUTHORS]: I prefer `AMiT Kumar`.

Comment (certik) [inline AUTHORS]: Just fix that in mailmap and regenerate AUTHORS.

**Comment by shivamvats** [comment]
+1
Thanks!

**Comment by cbehan** [comment]
+1 in case you're waiting for me.

**Pull request merged**